glide.lock

```diff
@@ -490,4 +490,14 @@ imports:
   - third_party/forked/golang/netutil
   - third_party/forked/golang/reflect
   - third_party/forked/golang/template
-testImports: []
+testImports:
```

Comment:
@adamreese nope. these test imports are dependencies of dependencies of helm. No need for them in the `glide.yaml` as they are transitive.